pkg/spoke/submarineragent/config_controller.go

```diff
@@ -35,6 +35,7 @@ import (
 	"k8s.io/client-go/kubernetes"
 	corev1lister "k8s.io/client-go/listers/core/v1"
 	"k8s.io/client-go/util/retry"
+	"k8s.io/klog"
```

Comment:
We use "k8s.io/klog/v2" at other places in the code. Can we use the same even here instead of adding new dependency on klog?

Reply:
Sure, it was auto-imported, I updated it to v2